### ISSUE TITLE
Is singlehtml supported by sphinx-a4doc?

### ISSUE THREAD

**Issue by godojoe** [taminomara/sphinx-a4doc#15]
I run sphinx-build with the  `-b singlehtml` option and get the following exception:
```
Exception occurred:
  File "c:\devtools\python3\lib\site-packages\docutils\nodes.py", line 1056, in copy
    obj = self.__class__(rawsource=self.rawsource, **self.attributes)
TypeError: __init__() got an unexpected keyword argument 'rawsource'
```

My python dependency tree looks like this:
```
pipdeptree -p sphinx-a4doc                                                                              
Warning!!! Possibly conflicting dependencies found:
* pytest==7.1.2
 - iniconfig [required: Any, installed: ?]
------------------------------------------------------------------------
sphinx-a4doc==1.3.0
  - antlr4-python3-runtime [required: ==4.7.1, installed: 4.7.1]
  - PyYAML [required: Any, installed: 3.11]
  - sphinx [required: >=1.8.0, installed: 6.0.0b2]
    - alabaster [required: >=0.7,<0.8, installed: 0.7.12]
    - babel [required: >=2.9, installed: 2.11.0]
      - pytz [required: >=2015.7, installed: 2022.1]
    - colorama [required: >=0.4.5, installed: 0.4.5]
    - docutils [required: >=0.18,<0.20, installed: 0.19]
    - imagesize [required: >=1.3, installed: 1.4.1]
    - importlib-metadata [required: >=4.8, installed: 5.1.0]
      - zipp [required: >=0.5, installed: 3.8.1]
    - Jinja2 [required: >=3.0, installed: 3.1.2]
      - MarkupSafe [required: >=2.0, installed: 2.0.0]
    - packaging [required: >=21.0, installed: 21.3]
      - pyparsing [required: >=2.0.2,!=3.0.5, installed: 3.0.9]
    - Pygments [required: >=2.12, installed: 2.12.0]
    - requests [required: >=2.5.0, installed: 2.22.0]
      - certifi [required: >=2017.4.17, installed: 2022.6.15]
      - chardet [required: >=3.0.2,<3.1.0, installed: 3.0.4]
      - idna [required: >=2.5,<2.9, installed: 2.8]
      - urllib3 [required: >=1.21.1,<1.26,!=1.25.1,!=1.25.0, installed: 1.25.11]
    - snowballstemmer [required: >=2.0, installed: 2.2.0]
    - sphinxcontrib-applehelp [required: Any, installed: 1.0.2]
    - sphinxcontrib-devhelp [required: Any, installed: 1.0.2]
    - sphinxcontrib-htmlhelp [required: >=2.0.0, installed: 2.0.0]
    - sphinxcontrib-jsmath [required: Any, installed: 1.0.1]
    - sphinxcontrib-qthelp [required: Any, installed: 1.0.3]
    - sphinxcontrib-serializinghtml [required: >=1.1.5, installed: 1.1.5]
```

My conf.py looks like this:

```
# Configuration file for the Sphinx documentation builder.
#
# For the full list of built-in configuration values, see the documentation:
# https://www.sphinx-doc.org/en/master/usage/configuration.html

# -- Project information -----------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information

project = 'Foo'
copyright = '2022, Foo'
author = 'Bar'

# -- General configuration ---------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration

extensions = [
    'sphinx_a4doc',
]

templates_path = ['_templates']
exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']



# -- Options for HTML output -------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output

html_theme = 'alabaster'

html_theme_options = {
    # Disable showing the sidebar. Defaults to 'false'
    'nosidebar': True,
}

html_static_path = ['_static']

from os.path import dirname
a4_base_path = dirname(__file__)
```

**Comment by taminomara** [comment]
Hi, I've released `1.4.0` with a fix. It should be available at PyPi in a few minutes.

**Comment by godojoe** [comment]
thank you! Fixed :-)
